### PR TITLE
Update tabs and accordion components to use simpler focus state

### PR DESCRIPTION
**Pull request by hannalaakso**
As per #1424, iterate on accordion and tabs focus states which we found were confusing for some users. This follows work done by @dashouse in #1394 to simplify the states.

## Accordion

<details>
<summary>IE8</summary>

ie8-tabs

</details>

<details>
<summary>IE11</summary>

ie11-tabs

</details>

<details>
<summary>Edge 18</summary>

edge-tabs

</details>

<details>
<summary>Firefox with changed colours</summary>

ff-tabs

</details>

<details>
<summary>Safari 12</summary>

safari-tabs

</details>

<details>
<summary>Chrome 75 (Mac)</summary>

chrome-tabs

</details>

<details>
<summary>Chrome (Galaxy S9) </summary>

chrome-mobile-tabs

</details>

## Tabs

<details>
<summary>IE8</summary>

acc-ie8

</details>

<details>
<summary>IE11</summary>

acc-ie11

</details>

<details>
<summary>Edge 18</summary>

tabs-edge-18

</details>

<details>
<summary>Firefox with changed colours</summary>

tabs-ff

</details>

<details>
<summary>Safari 12</summary>

tabs-safari

</details>

<details>
<summary>Chrome 75 (Mac)</summary>

acc-chrome

</details>

<details>
<summary>Chrome (Galaxy S9) </summary>

acc-chrome-mobile

</details>

Note: tabs don't have a hover state - discussed this with @dashouse, we might wish to review it together with https://github.com/alphagov/govuk-frontend/issues/1417.

Fixes https://github.com/alphagov/govuk-frontend/issues/1424
Closes https://github.com/alphagov/govuk-frontend/pull/1394

### CODE REVIEW

File: src/components/accordion/_accordion.scss [hunk 1]
```diff
@@ -197,13 +161,6 @@
       }
     }
 
-    // The accordion component has two focus state depending on if a section is expanded or closed.
-    // We also want to avoid styling when the secttion is pressed or `:active`
-    // to avoid the different focus styles from flashing quickly while the user interacts with the section.
-    .govuk-accordion__section:not(.govuk-accordion__section--expanded) .govuk-accordion__section-button:focus:not(:active) {
```
Comment (nickcolley) [inline src/components/accordion/_accordion.scss]: Goodbye my long and confusing friend

File: src/components/tabs/_tabs.scss [hunk 2]
```diff
@@ -91,56 +90,51 @@
...
 
-          @include govuk-if-ie8 {
-            border-top-color: $govuk-link-colour;
-          }
-
           &:focus {
-            box-shadow: inset 0 4px 0 0 $govuk-focus-colour, inset 0 8px 0 0 $govuk-focus-text-colour;
+
+            &:after {
+              // Add "higlight" on focused link
```
Comment (nickcolley) [inline src/components/tabs/_tabs.scss]: Non-blocking, typo 'higlight'

File: src/components/tabs/_tabs.scss [hunk 3]
```diff
@@ -91,56 +90,51 @@
...
           text-decoration: none;
 
-          @include govuk-if-ie8 {
-            border-top-color: $govuk-link-colour;
-          }
-
           &:focus {
-            box-shadow: inset 0 4px 0 0 $govuk-focus-colour, inset 0 8px 0 0 $govuk-focus-text-colour;
+
+            &:after {
```
Comment (nickcolley) [inline src/components/tabs/_tabs.scss]: Clever way of avoiding having to change the markup... 👍

**Comment by dashouse** [comment]
Looking really good.

Looks like we have lost the focus state on the no-js / mobile tab links though.

**Comment by hannalaakso** [comment]
Great spot @dashouse. The no-js / mobile tab link focus now fixed 👍

**Pull request merged**